src/components/nodes/sequenceFlow/sequenceFlow.vue

```diff
@@ -130,9 +130,9 @@ export default {
     targetIsNotSource() {
       return this.targetNode.id !== this.sourceNode.id;
     },
-    allowOutgoingFlow() {
-      return this.sourceConfig.allowOutgoingFlow == null ||
-        this.sourceConfig.allowOutgoingFlow(this.targetNode);
+    allowOutgoingSequenceFlow() {
```

Comment:
This bit was confusing for me initially. `allowOutgoingFlow` didn't actually seem to be defined in any of the element configs (not in modeler, PM core, or any one of the connectors); so it wasn't actually doing anything. Because of that I decided it was safe to re-name it to something more accurate and useful.